docs/instrument.md

```diff
@@ -20,15 +20,17 @@ For example, `nyc instrument . ./output` will produce instrumented versions of a
 
 The `--delete` option will remove the existing output directory before instrumenting.
 
+The '--in-place' option will allow you to run the instrument command
```

Comment:
End with a `.`?

Reply:
I also had single quotes instead of back ticks... eek.